app/src/features/surveys/observations/measurements/search/MeasurementsSearchAutocomplete.tsx

```diff
@@ -0,0 +1,204 @@
+import { mdiMagnify } from '@mdi/js';
+import Icon from '@mdi/react';
+import CheckBox from '@mui/icons-material/CheckBox';
+import CheckBoxOutlineBlank from '@mui/icons-material/CheckBoxOutlineBlank';
+import Autocomplete from '@mui/material/Autocomplete';
+import Box from '@mui/material/Box';
+import Checkbox from '@mui/material/Checkbox';
+import ListItem from '@mui/material/ListItem';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { CBMeasurementType } from 'interfaces/useCritterApi.interface';
+import { debounce } from 'lodash-es';
+import { useMemo, useState } from 'react';
+
+export interface IMeasurementsSearchAutocompleteProps {
+  /**
+   * The selected measurements.
+   *
+   * @type {CBMeasurementType[]}
+   * @memberof IMeasurementsSearchAutocompleteProps
+   */
+  selectedOptions: CBMeasurementType[];
+  /**
+   * An async function that returns an array of options, based on the provided input value.
+   *
+   * @memberof IMeasurementsSearchAutocompleteProps
+   */
+  getOptions: (inputValue: string) => Promise<CBMeasurementType[]>;
+  /**
+   * Callback fired on selecting options.
+   *
+   * Note: this is not fired until the user un-focuses the component.
+   *
+   * @memberof IMeasurementsSearchAutocompleteProps
+   */
+  onSelectOptions: (measurements: CBMeasurementType[]) => void;
+}
+
+/**
+ * Renders a search input to find and add measurements.
+ *
+ * @param {IMeasurementsSearchAutocompleteProps} props
+ * @return {*}
+ */
+const MeasurementsSearchAutocomplete = (props: IMeasurementsSearchAutocompleteProps) => {
+  const { selectedOptions, getOptions, onSelectOptions } = props;
+
+  const [inputValue, setInputValue] = useState('');
+  const [options, setOptions] = useState<CBMeasurementType[]>([]);
+
+  const [pendingSelectedOptions, setPendingSelectedOptions] = useState<CBMeasurementType[]>([]);
+
+  const handleSearch = useMemo(
+    () =>
+      debounce(async (inputValue: string, callback: (searchedValues: CBMeasurementType[]) => void) => {
+        const response = await getOptions(inputValue);
+        callback(response);
+      }, 500),
+    [getOptions]
+  );
+
+  return (
+    <Autocomplete
+      id="measurements-autocomplete"
+      data-testid="measurements-autocomplete"
+      noOptionsText="No matching options"
+      autoHighlight={true}
+      options={options}
+      multiple={true}
+      disableCloseOnSelect={true}
+      blurOnSelect={false}
+      clearOnBlur={false}
+      getOptionLabel={(option) => option.measurement_name}
+      //   getOptionLabel={(option) => option.commonName || option.scientificName}
```

Comment:
Removed.

Reply:
As as above: 

This is code that we do want, that Jer wrote, but which can't work just yet because the response from Critterbase is in flux. Ideally this, or some version of it, will be uncommented soon and the typography above will be removed.